.goreleaser.yml

```diff
@@ -23,30 +23,35 @@ archives:
       - goos: windows
         format: zip
     files:
-      - none*  # only package the binary - not defaults: readme, license, changelog
+      - README.md
+      - LICENSE
 
 checksum:
   name_template: 'SHA256SUMS.txt'
 
 brews:
   - description: Terraform provider for CyberArk Conjur
     homepage: https://github.com/cyberark/terraform-provider-conjur
-    url_template: https://github.com/cyberark/terraform-provider-conjur/releases/download/v{{.Version}}/terraform-provider-conjur-darwin-amd64.tar.gz
+    url_template: https://github.com/cyberark/terraform-provider-conjur/releases/download/v{{.Version}}/{{.ProjectName}}-{{.Version}}-{{.Os}}-{{.Arch}}.gz
```

Comment:
not a big deal, but should this be `v{{.Version}}` too?

Reply:
Hmm... you make a good point though it's just changing it to what the artifacts are created as. I may defer this to a later point since I have a few commits that are built upon this logic if that works for you.